engine/src/main/java/org/terasology/rendering/world/WorldRenderer.java

```diff
@@ -58,6 +58,11 @@
     void onChunkUnloaded(Vector3i chunkPos);
 
     /**
+     * Request a refresh of the render task list. The refresh takes place before the next frame.
+     */
+    void requestRenderTaskListRefresh();
```

Comment:
Please rename to requestTaskListRefresh(): there are no plans for additional task lists within the renderer and as such the term task list does not require disambiguation.